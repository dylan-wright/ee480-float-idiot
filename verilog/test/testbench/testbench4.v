op = `OPaddf;
x = 16'h4040;
y = 16'h4040;
#1 $display("%h", z);
x = 16'h4120;
y = 16'h41a0;
#1 $display("%h", z);
x = 16'h0000;
y = 16'h0000;
#1 $display("%h", z);
x = 16'h0000;
y = 16'h3f80; //1
#1 $display("%h", z);
x = 16'h4040;
#1 $display("%h", z);
x = 16'h4120;
#1 $display("%h", z);
