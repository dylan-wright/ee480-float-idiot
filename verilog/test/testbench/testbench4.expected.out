40c0
41f0
0000
3f80
4080
4130
0000
c4a7
c4a4
